test/e2e/specs/default-config/recordedit/multi-col-types.spec.js

```diff
@@ -118,9 +118,16 @@ describe('When editing a record', function() {
     // Tests that check the values for regular, non-disabled input fields are in 01-recordedit.edit.spec.js
     it('should display the correct values in disabled input fields', function(done) {
         testParams.table_w_generated_columns.row.forEach(function checkInput(col) {
-            var input = recordEditPage.getInputById(0, col.name);
-            expect(input.isEnabled()).toBeFalsy("col " + col.name + " was not disabled.");
-            expect(input.getAttribute('value')).toBe(col.value, "col " + col.name + " value missmatch.");
+            if (col.name == "asset_col_gen") {
```

Comment:
This is not really important and is just an observation. Since in the code the displayType of  disabled assets is not "disabled" anymore. You could modify the `displayType` of this column to be `upload`. And then in here instead of checking for the `col.name`, you could check its `col.displayType`.